CHANGELOG.md

```diff
@@ -23,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `VideoPriorityBasedPolicyConfig` to control video downlink policy with network event response and recovery delays. Check [User Guide for Priority-based Downlink Policy](https://aws.github.io/amazon-chime-sdk-js/modules/prioritybased_downlink_policy.html#user-guide-for-priority-based-downlink-policy) for more information.
 - Amazon Chime SDK Project Board Overview and Guide.
 - Added 25 video tile support for demo app.
+- Exposed Amazon Voice Focus model complexity as a type in order to support limiting complexity in the meeting demo.
```

Comment:
Could we please move this up in the [Unreleased] section?

Reply:
Ah, rebase!